### PR TITLE
[Docs] jdbc external table related changes in main

### PR DESCRIPTION
**Pull request by hellolilyliuyi**
## What type of PR is this：
- [ ] bug
- [ ] feature
- [ ] enhancement
- [ ] refactor
- [x] others

## Which issues of this PR fixes ：

Fixes #

### CODE REVIEW

**Review by amber-create — DISMISSED** [review]

File: docs/using_starrocks/External_table.md [hunk 1]
```diff
@@ -232,11 +232,11 @@ select * from es_table where esquery(k4, ' {
 
 ## External table for a JDBC-compatible database
 
-From v2.3.0, StarRocks provides external tables to query JDBC-compatible databases. This way, you can analyze the data of such databases in a blazing fast manner without the need to import the data into StarRocks.  This topic describes how to create an external table in StarRocks and query data in JDBC-compatible databases.
+From v2.3.0, StarRocks provides external tables to query JDBC-compatible databases. This way, you can analyze the data of such databases in a blazing fast manner without the need to import the data into StarRocks. This topic describes how to create an external table in StarRocks and query data in JDBC-compatible databases.
```
Comment (evelynzhaojie) [inline docs/using_starrocks/External_table.md:235]: this topic should be "this section"

File: docs/using_starrocks/External_table.md [hunk 2]
```diff
@@ -232,11 +232,11 @@ select * from es_table where esquery(k4, ' {
...
 
 ### Prerequisites
 
-When a JDBC resource is created, and a JDBC external table is queried for the first time, the FEs and BEs need to download the JDBC driver. Therefore, the machines on which the FEs and BEs are located must have access to the download URL of the JDBC driver during these two phases. The download URL is specified by the `driver_url` parameter when the JDBC resource is created.
+When a JDBC resource is created, and a JDBC external table is queried for the first time, the FEs and BEs need to download the JDBC driver. Therefore, the machines on which the FEs and BEs are located must have access to the download URL of the JDBC driver JAR package during these two phases. The download URL is specified by the `driver_url` parameter when the JDBC resource is created.
```
Comment (evelynzhaojie) [inline docs/using_starrocks/External_table.md:239]: When you create a JDBC resource or use a JDBC external table to query data for the first time, the FEs and BEs need to download the JDBC driver. Therefore, the machines on which the FEs and BEs are deployed must have access to the download URL of the JDBC driver. The download URL is specified by the `driver_url` parameter in the statement used for creating the JDBC resource.

File: docs/using_starrocks/External_table.md [hunk 3]
```diff
@@ -270,9 +270,9 @@ The required parameters in `properties` are as follows:
 
 > Note: The URI must include the name of the target database. For example, in the preceding code example, `jdbc_test` is the name of the target database that you want to connect.
 
-* `driver_url`:  the download URL of the JDBC driver used by the target database.
+* `driver_url`:  the download URL of the JDBC driver JAR package. An HTTP URL or file URL is supported. For example, `https://repo1.maven.org/maven2/org/postgresql/postgresql/42.3.3/postgresql-42.3.3.jar` or `file:///home/disk1/postgresql-42.3.3.jar`.
```
Comment (evelynzhaojie) [inline docs/using_starrocks/External_table.md:273]: for example后面不是完整句的话，不能独立成句子的，An HTTP URL or file URL is supported, for example, xxx

File: docs/using_starrocks/External_table.md [hunk 4]
```diff
@@ -270,9 +270,9 @@ The required parameters in `properties` are as follows:
...
 
-* `driver_class`: the class name of the JDBC driver used by the target database. The class names of common xxx are as follows:
+* `driver_class`: the class name of the JDBC driver. The class names of common xxx are as follows:
```
Comment (evelynzhaojie) [inline docs/using_starrocks/External_table.md]: XXX 改为 The driver class names of common databases are as follows:

**Review by evelynzhaojie — DISMISSED** [review]

**Review by evelynzhaojie — APPROVED** [review]

**Review by EsoragotoSpirit — APPROVED** [review]

**Comment by wanpengfei-git** [comment]
run starrocks_admit_test

**Comment by hellolilyliuyi** [comment]
run starrocks_admit_test

**Pull request merged**